api/src/models/user.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Schema representing a H4I member
+ */
+const mongoose = require('mongoose');
+
+const User = new mongoose.Schema({
+  classes: {
+    type: [String],
```

Comment:
Add a `default` field for the default value a user's `classes` property should be if they've added no classes.